web/pom.xml

```diff
@@ -852,7 +852,7 @@
             catalog/lib/bootstrap-table/docs/**
           </packagingExcludes>
           <!--          <warSourceDirectory>src/main/geonetwork</warSourceDirectory> -->
-          <webXml>${build.webapp.resources}/WEB-INF/web.xml</webXml>
+          <webXml>${build.webapp.resources}/geonetwork/WEB-INF/web.xml</webXml>
```

Comment:
We probably have to also change webApp.descriptor below in Jetty plugin ? Testing locally

Reply:
And travis fails with 

```

[ERROR] Failed to execute goal org.apache.maven.plugins:maven-war-plugin:3.2.3:war (default-war) on project web-app: The specified web.xml file '/home/travis/build/geonetwork/core-geonetwork/web/target/webapp/geonetwork/WEB-INF/web.xml' does not exist -> [Help 1]
[ERROR] 
```